repo/packages/B/bookkeeper/1/marathon.json.mustache

```diff
@@ -8,26 +8,41 @@
     "type": "DOCKER",
     "volumes": [
       {
-        "containerPath": "/bk/journal",
-        "hostPath": "/mnt/journal",
+        "containerPath": "/bookkeeper/data/journal",
```

Comment:
shouldn't this be parameterized instead of hard coded?

Reply:
👍, Thanks, will change it.